app.js

```diff
@@ -0,0 +1,45 @@
+function submitToAPI(e) {
+    e.preventDefault();
+
+    const phoneVal = document.getElementById("phone-input").value;
+    const topicVal = document.getElementById("topic-select").value;
+
+    const tel_regex = /[0-9]{10}/;
+    if (tel_regex.test(phoneVal)) {
+        var full_phone = '+1';
+        full_phone += phoneVal;
+        let data = {
+            number: full_phone,
+            topic: topicVal
+        };
+
+        $.ajax({
+            type: "POST",
+            url: "https://djjqpcoffj.execute-api.us-west-2.amazonaws.com/prod/",
+            dataType: "json",
+            crossDomain: "true",
+            contentType: "application/json; charset=utf-8",
+            data: JSON.stringify(data),
+
+            success: function () {
+                // clear form and show a success message
+                alert("Success!");
+                resetFields();
+            },
+            error: function () {
+                // show an error message
+                alert("Unsuccessful");
+                resetFields();
+            }
+        });
+    } else {
+        alert("Please enter a valid mobile number");
+        resetFields();
+    }
+}
+
+function resetFields() {
+    document.getElementById("phone-input").reset();
```

Comment:
![reset_notafunction](https://user-images.githubusercontent.com/29528527/76013552-eebe2700-5ecc-11ea-9b6e-424bbc5dfd94.png)

Calling `reset()` is not available for [textarea or DOM input elements](https://stackoverflow.com/a/37511171); following recommendation in that answer to clear the field's value. Note that this `reset()` function works fine for the radio buttons, just not the `input` field which isn't contained in a `form` field. (Welcome to JavaScript insanity.)
```suggestion
    document.getElementById("phone-input").value = "";
```